### PR TITLE
Add simple resources

### PR DESCRIPTION
**Pull request by pxlrbt**
This PR will add examples for Simple Resources to blog authors and blog categories.

### CODE REVIEW

File: app/Filament/Resources/Blog/AuthorResource/Pages/ListAuthors.php [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 class ListAuthors extends ListRecords
```
Comment (danharrin) [inline app/Filament/Resources/Blog/AuthorResource/Pages/ListAuthors.php]: The List pages need to be Manage pages, you can remove the traits & the other pages

**Review by danharrin — CHANGES_REQUESTED** [review]

File: app/Filament/Resources/Blog/AuthorResource/Pages/ListAuthors.php (hunk 1 above)
Comment (pxlrbt) [reply app/Filament/Resources/Blog/AuthorResource/Pages/ListAuthors.php]: ManagePages also includes `CanDeleteRecords`, but I guess it's fine to add this to the demo too

**Comment by danharrin** [comment]
Delete the old pages?

**Comment by pxlrbt** [comment]
> Delete the old pages?

Sorry, didn't get you right last time. Fixed it with https://github.com/laravel-filament/demo/pull/21

**Pull request merged**